system-variables.md

```diff
@@ -2083,6 +2429,14 @@ explain select * from t where age=5;
 
 ### tidb_restricted_read_only <span class="version-mark">New in v5.2.0</span>
 
+<CustomContent platform="tidb-cloud">
+
+> **Note:**
+>
+> This TiDB variable is not applicable to TiDB Cloud.
```

Comment:
This is inconsistent with the v6.1 version: https://github.com/pingcap/docs/pull/9479/files#diff-ec2cbb115cc58fe3afe2862eb5148732d323cbf7ae35dea97bd381a1f8d0e8a0R2288-R2292

Reply:
This is because we have made further changes in https://github.com/pingcap/docs/pull/9806.